app/renderer/reducers/frameReducer.js

```diff
@@ -92,7 +92,7 @@ const frameReducer = (state, action, immutableAction) => {
         }
       }
 
-      const title = immutableAction.getIn(['changeInfo', 'title'])
+      const title = immutableAction.getIn(['tabValue', 'title'])
```

Comment:
this may help, but won't be a complete fix because it doesn't address the race condition. Getting the title from `tabValue` will only fix the title when the tab state changes for some other reason. I think this change is ok for now, but we should add a comment/TODO that the race condition with appState is the real problem